Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/branch.py

```diff
@@ -99,6 +99,35 @@ def to_branch_name(cls, value):
                 result += c
         return string_utils.encode(result, target_type=str)
 
+    @classmethod
+    def create_radar(cls, args, repository, issue):
```

Comment:
Should name this `cc_radar` or even `cced_radar` to indicate it will _return_ the cced radar.